pkg/kubelet/dockertools/manager.go

```diff
@@ -579,7 +579,7 @@ func (dm *DockerManager) runContainer(
 		}
 	}
 	memoryLimit := container.Resources.Limits.Memory().Value()
-	cpuShares := milliCPUToShares(container.Resources.Limits.Cpu().MilliValue())
+	cpuShares := milliCPUToShares(container.Resources.Requests.Cpu().MilliValue())
```

Comment:
The PR to change the meaning of limits should be merged before the PR that introduces new support for requests IMHO.